language/bytecode_verifier/src/type_memory_safety.rs

```diff
@@ -58,7 +61,17 @@ impl<'a> TypeAndMemorySafetyAnalysis<'a> {
                 );
             }
         }
-        let initial_state = AbstractState::new(locals);
+        let module_view = ModuleView::new(module);
+        let mut globals = BTreeMap::new();
+        for (struct_definition_idx, struct_definition_view) in module_view.structs().enumerate() {
```

Comment:
Do we need to do this insertion now? You could just use `entry()` when you access it from the map

Reply:
I prefer to leave the code as is to avoid taxing my mind with extraneous invariants.